src/plugins/discover/public/utils/use_discover_services.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { useKibana } from '../../../kibana_react/public';
+import { DiscoverServices } from '../build_services';
+
+export const useDiscoverServices = () => useKibana<DiscoverServices>();
```

Comment:
AFAIK we're just using `.services` of `useKibana`, since the hook is also named `useDiscoverServices`, we could just return `services` of `useKibana` here. This would further simplify the code , WDYT?

Reply:
You are right, nice simplification.

Reply:
another simplification of this great simplification PR 🥳